pkg/distributor/distributor_test.go

```diff
@@ -7416,46 +7481,43 @@ func TestStartFinishRequest(t *testing.T) {
 	}
 }
 
-func TestSendMessageMetadata(t *testing.T) {
```

Comment:
Note to reviewers: this was was directly calling `Distributor.sendToIngester()` which doesn't exist anymore. I've refactored the test to run like other tests, and then assert on the gRPC context when the `mockIngester.Push()` request is received.